filament/src/materials/blitLow.mat

```diff
@@ -22,7 +22,9 @@ material {
     ],
     depthWrite : false,
     depthCulling : false,
-    domain: postprocess
+    domain: postprocess,
+    shadingModel : unlit,
```

Comment:
ShadingModel should be irrelevant for a `postprocess` domain material. We shouldn't have to specify it. I think internally we can just make it `unlit` automatically.

Reply:
Alright, I'll take care of this tomorrow.

Reply:
Done in the latest commit.